api/src/main/webapp/WEB-INF/appengine-web.xml.template

```diff
@@ -16,6 +16,9 @@
     <!-- NOTE(RW-235) Keep the db connection alive. -->
     <env-var name="spring.datasource.dbcp.test-on-borrow" value="true" />
     <env-var name="spring.datasource.dbcp.validationQuery" value="SELECT 1" />
+    <env-var name="spring.datasource.dbcp.test-while-idle" value="true" />
+    <env-var name="spring.datasource.dbcp.time-between-eviction-runs-millis" value="3600000" />
+    <env-var name="spring.datasource.dbcp.validation-query" value="SELECT 1" />
```

Comment:
Is this a different setting than validationQuery above?

Reply:
Whoops, I didn't notice it was the same for `test-on-borrow` and `test-when-idle`. Removed the dup.

Reply:
So... before you had validationQuery. Now you have validation-query. I assume it's supposed to be the latter?

Reply:
Yes; from comments on SO I glean that these properties switched from camel case to hyphenated at some point.